packages/ra-supabase-core/README.md

```diff
@@ -0,0 +1,222 @@
+# ra-supabase-core
+
+This package provide a dataProvider, an authProvider and hooks to integrate [Supabase](https://supabase.io/) with [react-admin](https://marmelab.com/react-admin).
+
+## Usage
+
+```jsx
+// in supabase.js
+import { createClient } from '@supabase/supabase-js';
+
+export const supabase = createClient(
+    SUPABASE_URL,
+    SUPABASE_ANON_KEY
+);
+
+// in dataProvider.js
+import { supabaseDataProvider } from 'ra-supabase-core';
+import { supabase } from './supabase';
+
+const resources = {
+    posts: ['id', 'title', 'body', 'author_id', 'date'],
+    authors: ['id', 'full_name'],
+}
+
+export const dataProvider = supabaseDataProvider(supabase, resources);
+
+// in authProvider.js
+import { supabaseAuthProvider } from 'ra-supabase-core';
+import { supabase } from './supabase';
+
+export const authProvider = supabaseAuthProvider(supabase, {
+    getIdentity: (user) => {
+        const { data, error } = await supabase
+            .from('userProfiles')
+            .select('id, first_name, last_name')
+            .match({ email: user.email })
+            .single();
+
+        if (!data || error) {
+            throw new Error();
+        }
+
+        return {
+            id: data.id,
+            fullName: `${data.first_name} ${data.last_name}`
+        };
+    }
+});
+
+// in App.js
+import { Admin, Resource, ListGuesser } from 'react-admin';
+import { dataProvider } from './dataProvider';
+import { authProvider } from './authProvider';
+
+export const MyAdmin = () => (
+    <Admin
+        dataProvider={dataProvider}
+        authProvider={authProvider}
+    >
+        <Resource name="posts" list={ListGuesser} />
+        <Resource name="authors" list={ListGuesser} />
+    </Admin>
+)
+```
+
+## Authentication
+
+We currently only supports email/password authentication. The `supabaseAuthProvider` provides the `setPassword` method in addition of the `authProvider` methods required by `react-admin`. This method allows you to create UI for users to set their passwords after being invited for example. This could be done in a custom route.
```

Comment:
```suggestion
We currently only support email/password authentication. The `supabaseAuthProvider` provides the `setPassword` method in addition of the `authProvider` methods required by `react-admin`. This method allows you to create UI for users to set their passwords after being invited for example. This could be done in a custom route.
```